mir_eval/key.py

```diff
@@ -124,6 +138,11 @@ def weighted_score(reference_key, estimated_key):
     # If keys are the same, return 1.
     if reference_key == estimated_key and reference_mode == estimated_mode:
         return 1.
+    # If reference or estimated key are x and they are not the same key
+    # then the result is 'Other'.
+    if reference_key == 'X' or estimated_key == 'X' \
+            or reference_key == 'x' or estimated_key == 'x':
```

Comment:
Why not just use .lower() when doing these checks?

Reply:
mmm you are right!!!